prow/cluster/plank_deployment.yaml

```diff
@@ -29,13 +29,22 @@ spec:
         role: prow
       containers:
       - name: plank
-        image: gcr.io/k8s-prow/plank:0.17
+        image: gcr.io/k8s-prow/plank:0.18
+        args:
+        - "--dry-run=false"
+        - "--github-bot-name=k8s-ci-robot"
```

Comment:
> I would love to submit a follow-up PR and make the move if you guys are ok with it. I am more inclined to put it in the config but tying it with the oauth token in the secret is not unreasonable too.

SG. I think both make some sense.

[1]: I'd like to make that whole string templatable.
[2]: Both #2721 and #2566 take a stab at fixing, but I'm not super happy with either.